derive/src/pyclass.rs

```diff
@@ -42,6 +42,182 @@ enum ClassItem {
     },
 }
 
+struct ClassItemMeta<'a> {
+    sig: &'a Signature,
+    parent_type: &'static str,
+    meta: HashMap<String, Option<Lit>>,
+}
+
+impl<'a> ClassItemMeta<'a> {
+    const METHOD_NAMES: [&'static str; 2] = ["name", "magic"];
+    const PROPERTY_NAMES: [&'static str; 3] = ["name", "magic", "setter"];
```

Comment:
You can make these `const _NAMES: &'static [&'static str] = &[...];` so that you don't have to change the size in the type when you add a new one.